plugin/audit_log/tests/mtr/prepare_stmt_audit_log.py

```diff
@@ -0,0 +1,54 @@
+"""
+This script file executes prepared statements
+which will be used to test audit_log
+command_class field
+
+Example Usage (in Mysql Test Framework):
+
+	let $exec = python \
+	MYSQL_BASEDIR/plugin/audit_log/tests/mtr/prepare_stmt_audit_log.py \
+	root 127.0.0.1 $MASTER_MYPORT information_schema mysql
+
+"""
+
+import sys
+
+py_version = sys.version_info
+
+if len(py_version) > 0:
+	py_version = int(py_version[0])
+else:
+	py_version = 2
+
+if py_version == 2:
+	import MySQLdb
+else:
+	import mysql.connector
+
+if __name__ == '__main__':
+
+	if len(sys.argv) != 6:
+		print("Usage : prepare_stmt_audit_log.py <user> <host> <port> <db_name> <pass>")
+		sys.exit(1)
+
+	user = sys.argv[1]
+	host = sys.argv[2]
+	port = int(sys.argv[3])
+	db = sys.argv[4]
+	p = sys.argv[5]
+	cursor = None
+
+	if py_version == 3:
+		cnx = mysql.connector.connect(database=db, user=user,password=p,port=port)
+		cursor = cnx.cursor(prepared=True)
+		stmt = "SHOW GLOBAL VARIABLES WHERE Variable_name = ?"
```

Comment:
cant we use 
PREPARE stmt FROM SHOW GLOBAL VARIABLES WHERE Variable_name = ?;
SET @var2='blah';
EXECUTE insert_stmt USING @var2;

Do we need this python script ?

Reply:
These statements will generate Audit logs with name "Query" not name "Prepare".
But MySQLdb python driver used for python2 in mtr tests also generates "Query" logs .
So , I will remove python script and use these statements.